### ISSUE TITLE
python-libxdo-0.1.2a0 source dist on pypi is missing CHANGELOG.rst

### ISSUE THREAD

**Issue by mzizzi** [rshk/python-libxdo#16]
If you go to pypi and download the latest (0.1.2a0) archive you'll see that CHANGELOG.rst is missing.  This is strange because when running `make package` locally and checking the contents of the package it appears that CHANGELOG.rst is included.

**Comment by rshk** [comment]
Strange indeed, I'm going to have a look asap. Sorry for the inconvenience

**Comment by mzizzi** [comment]
Any chance we could get a fix + version bump in pypi for this?  Right now installing version 0.1.2a0 causes setup.py to error out.

```
$ pip install python-libxdo==0.1.2a0
Collecting python-libxdo==0.1.2a0
  Using cached python-libxdo-0.1.2a0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-hCONwV/python-libxdo/setup.py", line 13, in <module>
        with open(os.path.join(here, 'CHANGELOG.rst')) as fp:
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-hCONwV/python-libxdo/CHANGELOG.rst'
```

**Comment by rshk** [comment]
Fixed in 0.1.2a1. Sorry for the inconvenience, I forgot to add a MANIFEST.in
